### PR TITLE
ES:  CO Kyokuya

### PR DESCRIPTION
**Pull request by NozomiRem**

### CODE REVIEW

File: Orders/Kyokuya/co06033202.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
-Title#0,"""Desesperacion, mi extraña sombra"""
-Explanation#0,"""...Oh, eres tú otra vez. ¿Quieres descender tanto a la oscuridad?<br>Jejeje...luego muéstrame tu resolución. Desecha cualquier pensamiento tonto de justicia... ¿Estás dispuesto a hacer lo que sea necesario?<br>seré yo quien determine que...<br><br><yellow>※Solo Phantom<br>Objetivo: Enemigos Nv. 41+<c>"""
-Reward#0,"""+5 Puntos de Skill Phantom"""
+Title#0,"""Desesperacion, mi sombra"""
+Explanation#0,"""...Oh, eres tú otra vez.<br>¿Tanto deseas descender a la oscuridad?<br>Jejeje...luego muéstrame tu resolución.<br>Desecha cualquier pensamiento<br>ridiculo de justicia...<br>¿Estás dispuesto a hacer lo que sea necesario?<br>seré yo quien lo determine...<br><br><yellow>※Solo Phantom<br>Objetivo: Enemigos Nv. 41+<c>"""
+Reward#0,"""+5 Puntos de Skill Phantom"""
```
Comment (alama) [inline Orders/Kyokuya/co06033202.csv]: Why is the EOL missing on the Kyokuya COs?

Comment (TheBlazter) [reply Orders/Kyokuya/co06033202.csv]: Accidental delete fixing a conflict

File: Orders/Kyokuya/co06010005.csv [hunk 2]
```diff
@@ -1,3 +1,3 @@
-Title#0,"""ファントムクラス適性審査試験・Ⅳ"""
-Explanation#0,"""これは、個々のアークスにおける<br>ファントムクラスの適性を審査し<br>能力制限を解除するための試験である。<br>一層の能力向上を望む者の挑戦を推奨する。<br><yellow>※ファントム限定、パーティーを組まずに<br>抜剣でヴァーダーソーマを撃破<br>長銃でビオル・メデューナを撃破<br>長杖でグワナーダを撃破<br>Lv.61～のエネミーが対象<br>※ファントムのレベル上限を75まで解放<c>"""
-Reward#0,"""Phantom Licencia de Limite Nv. Ⅳ"""
+Title#0,"""Prueba de Aptitud IV de Clase Phantom"""
+Explanation#0,"""Esta es una prueba para determinar si un<br> ARKS es adecuado para la clase Phantom.<br>Esto te permitirá evitar la restricción de Lvl.<br>Al lograr el objetivo, es posible<br>aumentar su lvl.<br><yellow>※※ Sólo Phantom, Derrota  [Vardha Soma] <br>usando una Katana<br>Derrota [Biol Meduna] usando un Rifle<br>Derrota [Gwanahda] usando un Rod<br>enemigos Lv.61+<br> para aumentar el límite de nivel<br> de Phantom a 75.<c>"""
```
Comment (alama) [inline Orders/Kyokuya/co06010005.csv]: there a `NO-BREAK SPACE` character in between "Derrota" and "[Vardha Soma]"

Comment (TheBlazter) [reply Orders/Kyokuya/co06010005.csv]: Fixed

**Pull request merged**